site/public/templates/grading/Functions.twig

```diff
@@ -1,24 +1,19 @@
 {#
 Function for getting the badge style for a given set of total and earned points
 #}
-{% macro getBadgeStyle(earned, total, complete) %}
```

Comment:
I didn't realize there was more usages of this. It might actually be better to keep this complete variable and then set it to null in the one case where it's called where we don't define `ta_grading_complete` anymore. And then in here check if it's null rather than undefined. Unless you have tested and verified this macro still works as expected without that last parameter?

Reply:
That parameter doesn't appear to be used anywhere in the codebase.